doc/source/management/data_at_rest_encryption.rst

```diff
@@ -477,16 +461,81 @@ System Variables
 
 The variable turns on binary and relay logs encryption.
 
+.. rubric:: Redo log encryption
+
+Implemented in 8.0.16-7, the key rotation for the redo log has been redesigned to rotate the key with `SELECT rotate_system_key("percona_redo").
+
+Implemented in 8.0.16-7, the only supported values for `set global innodb_redo_log_encrypt` are the following:
 
+    * ON
+    * OFF
+    * master_key
+    * keyring_key 
+
+The changed page tracking may read pages which have the encryption bit set.  The redo log does not have the encryption key and decryption fails and generates an error message. A flag lets the read process safely ignore encryption errors in this case.
+
+If the server is started without keyring support attempting to encrypt the redo log with either the master_key option or the keyring_key option generates a warning in the error log. 
```

Comment:
This also isn't true: it's how upstream mysql works. We modified PS so it aborts startup with any incorrect encryption configuration.

* No keyring specified, redo log encryption requested: abort
* Keyring specified, but a different redo log encryption method specified (e.g. was master_key before, now it's keyring_key): abort

Reply:
After starting the server, an attempt to encrypt the redo log fails in the following conditions:

    * Server started with no keyring specified
    * Server started with a keyring, but a different redo log encryption method is specified